package.json

```diff
@@ -20,36 +20,47 @@
   },
   "preferGlobal": true,
   "bin": {
-    "labelcat": "cli.js"
+    "labelcat": "bin/labelcat.js"
   },
   "engines": {
     "node": ">= 8.x"
   },
   "scripts": {
     "cover": "nyc --reporter=lcov mocha test/*.js && nyc report",
-    "lint": "eslint src/ samples/ system-test/ test/",
-    "prettier": "prettier --write src/*.js src/*/*.js test/*.js test/*/*.js system-test/*.js system-test/*/*.js",
+    "lint": "eslint src/ system-test/ test/",
+    "prettier": "prettier --write src/*.js test/*.js system-test/*.js",
     "system-test": "mocha system-test/*.js --timeout 600000",
     "test-no-cover": "mocha test/*.js",
     "test": "npm run cover"
   },
   "dependencies": {
+    "chai": "^4.1.2",
     "@google-cloud/automl": "^0.1.2",
     "axios": "^0.18.0",
     "csv-write-stream": "^2.0.0",
     "json2csv": "^4.2.1",
+    "loglevel": "^1.6.1",
+    "mocha": "^5.2.0",
+    "proxyquire": "^2.1.0",
+    "sinon": "^6.3.4",
+    "sinon-chai": "^3.2.0",
```

Comment:
Remove `chai`, `mocha`, `proxyquire`, `sinon` and `sinon-chai` from the `dependencies` block (they're already in `devDependencies`, where they should be)

Reply:
When I had them in only devDependencies, I kept getting a "no-unpublished-require" issue when I ran the linter. Is there a better way to fix that?